### DISCUSSION TITLE
Only showing Amazon and Bestbuy

### DISCUSSION THREAD

**Discussion by Crow25** [jef/streetmerchant#2441] [Help]
Hello, I'm very new to this so sorry in advance. I'm trying to add EU stores for nvidia cards, I opened the dotenv with notepad and added the stores and list series, but when I type " npm run start " it always uses only bestbuy and amazon and series that I didn't add (ps5,xbox).
Is there something I can do to fix it?

**Comment by Mr-Tech-13** [comment]
What is the config file name? I misspelled mine. It should be dotenv

**Reply by Crow25** [reply]
Thank you,After a while I think i managed to fix it finding a folder called "streetmerchant-main"
